src/pods/toolbar/components/export-button/export-button.business.ts

```diff
@@ -28,6 +28,78 @@ export const calculateTableEndYPosition = (table: TableVm): number => {
   return table.y + tableHeight;
 };
 
+export const isTableOverlap = (table: TableVm, tables: TableVm[]): boolean => {
```

Comment:
In proper english, overlap is the verb, so rather:

```
doesTableOverlap
```

Reply:
done